### ISSUE TITLE
Ignoring when previous version doesn't exist

### ISSUE THREAD

**Issue by vmassol** [siom79/japicmp#122]
I'd like to ignore when the previous version defined doesn't exist. I know there's a `ignoreMissingClasses` property but it's defined as " Set this to true in order to ignore artifacts that cannot be resolved, i.e. the build does not break in case a dependency cannto be resolved to a file.". I want to break if some dependency of the current version cannot be resolved. I just don't want to break if the previous version doesn't exist without having to explicitly declare a skip, because if I do then I know for sure we'll forget to remove the skip when the next version is released.

Is that possible? If not, could it be added?

Thanks

**Comment by siom79** [comment]
Added new parameter `ignoreMissingOldVersion` to the maven plugin that won't break the build if the old version cannot be resolved.

If you like, you can try it on the `develop` branch:
```
git clone https://github.com/siom79/japicmp.git
cd japicmp
git checkout develop
mvn install
```

**Comment by siom79** [comment]
Released with 0.7.2.
